docs/LUA.md

```diff
@@ -41,8 +41,10 @@ Parameters which are not available are not displayed on the screen (i.e. the lis
 
 ## BW Lua Script Notes
 
-The Lua script for radios with a black and white screen is limited in functionality and only allows for 5 parameters to be configured.  By default, these are BindPhrase, Mode, TxPwr, RxPwr, and RxOutMode.
+The Lua script for radios with a black and white screen now also supports all parameters.  But because of screen and memory limitations it looks and behaves slightly differently than the color Lua script.
 
-If one wants to be able to change a different parameter, the 'custom param list' section in the Lua script can be updated to reference a different parameter. The parameter numbers are zero-based and can be determined from the 'Setup parameter list' section located in [setup_list.h](https://github.com/olliw42/mLRS/blob/main/mLRS/Common/setup_list.h).
+The starting screen page contains [common parameters](PARAMETERS.md#mlrs-documentation-parameters-v1300), status information, and commands such as bind and save.  Seven additional parameters appear on each subsequent page and page navigation is linear via the "prev" and "next" commands.
 
-For example, if one wanted to replace "Mode" with "RF Band" then 'param_idx_list[1] = 1' needs to be updated to 'param_idx_list[1] = 2'
+Parameters which are not available are still displayed, but with a "-" character in the value position.  Parameters which cannot be changed are displayed as normal.  Unused lines on the last screen page contain a blank name and a value of ".".  The cursor can be moved to unavailable or unchangeable parameters and unused lines, but they can not be selected for edit with enter.
```

Comment:
is there a haptic response?

Reply:
Yes, but I don't feel the need to document it

Reply:
I think it would be nice. It might be not immediately obvious to everyone what the meaning and pattern is.

Reply:
But since it seems you do, I did now.